tool/internal_pkg/pluginmode/thriftgo/hessian2.go

```diff
@@ -140,7 +141,11 @@ func loadIDLRefConfig(file *os.File) *config.RawConfig {
 	return idlRefCfg
 }
 
-var JavaObjectRe = regexp.MustCompile(`\*java\.Object\b`)
+var (
+	JavaObjectRe                     = regexp.MustCompile(`\*java\.Object\b`)
+	JavaExceptionRe                  = regexp.MustCompile(`\*java\.Exception\b`)
+	JavaExceptionEmptyVerificationRe = regexp.MustCompile(`return p\.Exception != nil\b`)
+)
 
 // ReplaceObject args is the arguments from command, subDirPath used for xx/xx/xx
```

Comment:
`ReplaceObject` is not good enough now. Maybe `hessianPatchByReplace`.

First letter better be lowercase since it's not used outside this package. Also for the patterns above, no need to be exported symbols.

Reply:
```ReplaceObject``` would be invoked by /tool/cmd/kitex/main.go.